.github/workflows/keycloak-scalability-benchmark.yml

```diff
@@ -181,10 +177,6 @@ jobs:
         with:
           project: ${{ env.PROJECT }}
 
-      - name: Scale down the cluster
```

Comment:
Would we still want this to be there? If the cluster decides to autoscale based on this scaling group?

Reply:
This step is not scaling down anything if the resources are still used. It is allowing the machinepool to scale to 0 if nodes are not needed. After this PR we are never setting min-replicas to > 0 so we don't need this step.